### PR TITLE
Minor notes on ICEM tutorial

### PR DESCRIPTION
**Pull request by hajdik**
## Purpose
@sseraj and I noticed a few things that would be helpful to note in the surface meshing portion of the tutorial.

## Expected time until merged
A few days (just minor changes)

## Type of change
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [X] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Checklist
- [ ] I have run `flake8` and `black` to make sure the code adheres to PEP-8 and is consistently formatted
- [ ] I have run unit and regression tests which pass locally with my changes
- [ ] I have added new tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

**Comment by nwu63** [comment]
I haven't meshed anything in ages, so I nominate @ArshSaja and @gang525 instead.

**Review by marcomangano — DISMISSED** [review]
Haven't played with ICEM for a bit but LGTM

File: machAeroTutorials/aero_icem.rst [hunk 1]
```diff
@@ -75,6 +77,7 @@ To adjust your view of the geometry in ICEM the following functions are possible
     - Drag mouse left/right: Rotate view in viewing plane
```
Comment (gang525) [inline machAeroTutorials/aero_icem.rst:77]: You can also drag the mouse while right clicking to do planar rotation of the viewing plane

**Review by gang525 — DISMISSED** [review]
Good stuff

**Review by sseraj — APPROVED** [review]

**Pull request merged**